src/Zend/Captcha/Word.php

```diff
@@ -357,7 +356,7 @@ public function generate()
 
     protected function _generateRandomId()
     {
-        return md5(Zend_Crypt_Math::randBytes(32));
+        return md5((string) Zend_Crypt_Math::randBytes(32));
```

Comment:
apparently the `randBytes` method can return `false`, this shouldn't change behavior though.